fuzz/fuzz_targets/deserialize_br.rs

```diff
@@ -0,0 +1,26 @@
+#![no_main]
+use clvmr::allocator::Allocator;
+use clvmr::node::Node;
+use clvmr::serde::node_from_bytes_backrefs;
+use clvmr::serde::node_to_bytes_backrefs;
+use libfuzzer_sys::fuzz_target;
+
+fuzz_target!(|data: &[u8]| {
+    let mut allocator = Allocator::new();
+    let program = match node_from_bytes_backrefs(&mut allocator, data) {
+        Err(_) => {
+            return;
+        }
+        Ok(r) => r,
```

Comment:
Since I actually found problems through fuzzing, it's definitely finding valid serialization. Note that if the serialization completes and there are still bytes remaining at the end, it passes, so that leniency helps a lot.

Reply:
Done in 31218ee4c5530fe5b2b69ee0f7ffb3286104cc55